src/App.vue

```diff
@@ -86,17 +113,32 @@ export default {
       if (this.stopPoint != null  || this.lineId != undefined) {
         getPlatforms(this.stopPoint)
           .then(response => {
-            this.timetable = response;
+            this.timetable = response
             
             const lineTimetable = this.timetable.filter(x => {
               return x.lineId === this.lineId
             });
 
-            this.platforms = [...new Set(lineTimetable.map(i => i.platformName))]
-              .sort((a, b) => (a[a.length - 1] > b[b.length - 1]) ? 1 : -1)
-          })
-          .catch(error => alert(error.name))
-      }      
+            const duplicatedPlatforms = lineTimetable.map(i => {
+              return { name: i.platformName, value: i.platformName }
+            })
+
+            this.platforms = new Dedupe(duplicatedPlatforms)
```

Comment:
I don't think this should have `new` in front of the function call. We're not instantiating a variable unless I'm missing something?

Reply:
Not sure why I did it that way, but I've removed it